### DISCUSSION TITLE
Welcome to geohey Discussions!

### DISCUSSION THREAD

**Discussion by giswqs** [giswqs/geohey#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TMD233** [comment]
When I trying to syn the change, the doc commit part always error with Exit code 65. it says Typo but I can not find the Error file. It says in egg-info folder and SOURCES.txt file.

**Comment by giswqs** [comment]
Please create a new discussion and inlcude link to the repo. Do not post questions here.
